### DISCUSSION TITLE
sponsorship question

### DISCUSSION THREAD

**Discussion by minsky91** [Acly/krita-ai-diffusion#687] [Q&A] [answered]
Hello, I am a digital AI artist as well as a sporadic software programmer. I would like to contribute financially to support development of this wonderful tool.  Is there a way to make a onetime sponsorship contribution above the $25 limit? (couldn't find an answer to this question in the discussion history)

**Comment by Acly** [comment, marked as answer]
Hi, and thank you for your support!
The $25 is only a suggestion not a limit, I believe the top-most option allows you to enter any custom amount you like.

**Comment by minsky91** [comment]
Oh yes indeed, thanks!
